draft-ietf-quic-recovery.md

```diff
@@ -763,24 +763,33 @@ The RECOMMENDED value is 2 * max_datagram_size.
 
 ## Slow Start
 
-While in slow start, QUIC increases the congestion window by the
-number of bytes acknowledged when each acknowledgment is processed, resulting
-in exponential growth of the congestion window.
+The congestion controller is in slow start any time the congestion window is
+below the slow start threshold.  The congestion controller begins in slow start.
```

Comment:
...because the ICW is less than SSThresh, presumably?  Might be worth mentioning that explicitly.

Reply:
+1 to this suggestion, since I had the same thought.  ssthresh is initialized to infinity, which is in the pseudocode, but not in this text I'm now noticing.
```suggestion
below the slow start threshold.  The congestion controller begins in slow start
because the slow start threshold is initialized to an infinite value.
```